src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    private static final int NUMBER_COUNT = 7;
+    private static final int FIGURE_COUNT = 5;
+
+    public int getRandomNumber() {
```

Comment:
```suggestion
    private int getRandomNumber() {
```

давай сделаем этот метод приватным и перенесем его в конец класса (приватные методы должны следовать за публичными)